apps/product/tasks.py

```
@@ -1,207 +1,982 @@
-        })
-        report_response = requests.post(yandex_market_sales_url, data=payload, headers=headers)
-        print(report_response.json())
-        report_id = report_response.json()['result']['reportId']
-        reponse = requests.get(yandex_report_url.format(report_id=report_id), headers=headers)
-        print(reponse.json())
-        while reponse.json()['result']['status'] == 'PROCESSING':
-            sleep(5)
-            reponse = requests.get(yandex_report_url.format(report_id=report_id), headers=headers)
-            print(reponse.json())
-        data = {}
+from json import dumps
```

Ruff

`json.dumps` imported but unused

Remove unused import: `json.dumps`

(F401)